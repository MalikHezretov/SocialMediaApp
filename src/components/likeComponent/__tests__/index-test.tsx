import 'react-native';
import React from 'react';
import LikeComponent from '..';
import renderer from 'react-test-renderer';

it('renders LikeComponent correctly', () => {
  const testRenderer = renderer.create(<LikeComponent numberOfLikes={20} />);
  const testInstance = testRenderer.root;

  expect(testInstance.findByType(LikeComponent).props.numberOfLikes).toBe(20);
  expect(testRenderer).toMatchSnapshot();
});
