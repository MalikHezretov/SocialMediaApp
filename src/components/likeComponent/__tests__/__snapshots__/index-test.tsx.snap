// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders LikeComponent correctly 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "marginBottom": 16,
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../src/common/assets/images/like.png",
      }
    }
    style={
      Object {
        "height": 25,
        "marginLeft": 32,
        "marginTop": 8,
        "width": 25,
      }
    }
  />
  <Text
    style={
      Object {
        "fontWeight": "700",
        "marginLeft": 4,
        "marginTop": 13,
      }
    }
  >
    20
  </Text>
</View>
`;
